hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/handlers/TopicHandler.java

```diff
@@ -47,6 +47,7 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
         onRequestValid(exchange, messageId, cachedTopic -> {
             exchange.addExchangeCompleteListener(new ExchangeMetrics(cachedTopic));
             exchange.putAttachment(AttachmentContent.KEY, new AttachmentContent(cachedTopic, new MessageState(), messageId));
+            exchange.setStatusCode(500);
```

Comment:
Few notes:
1. Please use here static field from `io.undertow.util.StatusCode.INTERNAL_SERVER_ERROR`
2. Consider moving this line to a dedicated private method with a name such as `setDefaultResponseCode`
3. Add a comment to this line (or method if you do 2.) why we are setting INTERNAL_SERVER_ERROR. It will help future developers and reviewers to understand it when they come across this piece of code.